google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -150,8 +150,12 @@ class TracingMessageBatch : public MessageBatch {
 
     // The first span in `batch_sink_spans` is the parent to the other spans in
     // the vector.
-    internal::OTelScope scope(batch_sink_spans.front());
-    return [oc = opentelemetry::context::RuntimeContext::GetCurrent(),
+    auto scope =
+        std::make_shared<internal::OTelScope>(batch_sink_spans.front());
+    // Capture the scope so it stays alive the returned function
```

Comment:
s/stays alive the/stays alive until the/

Reply:
done